File: core/src/me/game/sanxiao/SanXiao.java
package me.game.sanxiao;

import com.badlogic.gdx.graphics.g2d.SpriteBatch;
import game.kt.MyGame;

/**
 * Created by wangbo on 16/6/6.
 */
public class SanXiao extends MyGame {

	//TODO 点击会有框提示
	//TODO 第二次点击会交换
	//DO 没有可以消除的退回
	//TODO 4个和5个的特殊效果

	@Override
	public void create() {
		batch = new SpriteBatch();
		Assests.init();
		setScreen(new MainMenuScreen(this));
	}

	@Override
	public void render() {
		super.render();
	}
}

File: core/src/main/kotlin/SanXiao.kt
package game.kt

import com.badlogic.gdx.Game
import com.badlogic.gdx.graphics.g2d.SpriteBatch
import me.game.sanxiao.Assests

/**
 * Created by wangbo on 2016/10/20.
 */

abstract class MyGame() : Game() {
    lateinit var batch: SpriteBatch
}

class SanXiao : MyGame() {

    override fun create() {
        batch = SpriteBatch()
        Assests.init()
        setScreen(MainMenuScreen(this))
    }

    override fun render() = super.render()
}